addons/l10n_din5008/data/report_layout.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="report_layout_din5008" model="report.layout">
+            <field name="view_id" ref="l10n_din5008.external_layout_din5008"/>
+            <field name="image">/l10n_din5008/static/img/preview_din.png</field>
+            <field name="pdf">/l10n_din5008/static/pdf/preview_din.pdf</field>
+            <field name="name">DIN 5008</field>
+        </record>
+    </data>
+</odoo>
```

Comment:
did you `git move` those files from l10n_de? in github they appear as deleted and recreated (but we already had similar wrong info from github)

Reply:
I think the reason that it appears in the diff is that the file wasn't just moved.
Many lines were also modified to switch from `l10n_de` to `l10n_din5008`.